platform/core/src/utils/measurementGroupings.ts

```diff
@@ -0,0 +1,43 @@
+/**
+ * Groups measurements by study in order to allow display and saving by study
+ * @param {Object} servicesManager
+ */
+export const groupByStudy =
+  ({ servicesManager }) =>
+  items => {
+    const { displaySetService } = servicesManager.services;
+
+    const getItemStudyInstanceUID = item => {
+      const displaySet = displaySetService.getDisplaySetByUID(item.displaySetInstanceUID);
+      return displaySet.instances[0].StudyInstanceUID;
+    };
+
+    const studyInstanceUIDsSet = new Set(items.map(getItemStudyInstanceUID));
+    const uniqueStudyInstanceUIDs = Array.from(studyInstanceUIDsSet);
+
+    return uniqueStudyInstanceUIDs.map(studyInstanceUID =>
+      items.filter(item => {
+        const itemStudyInstanceUID = getItemStudyInstanceUID(item);
+        return itemStudyInstanceUID === studyInstanceUID;
+      })
+    );
+  };
+
+export const groupIntoSingleGroup = () => items => [items];
+
+export const groupByStudyDeprecated =
```

Comment:
Why is there a deprecated version?

Reply:
This is because I had to change this groupByStudy from the preivou PR... so I renamed it depecrated. But in the end this will be removed, since the changes in the original panel from last PR don't really make sense anymore.

So this won't be merged. I'll remove this in the end.